Client/Application/AppDelegate.swift

```diff
@@ -120,6 +120,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         // Ecosia
         FeatureManagement.fetchConfiguration()
         MMP.sendSession()
+        _ = TreeCounter.shared //Forcing statistics to be fetched before they are used anywhere
```

Comment:
The fact that by calling `shared` we are implicitly making a "fetch call" or any other business logic needs separation.
Was wondering where to make this even more explicit by making an explicit function that executes that under the hood. While it may sound like an over-engineering, for instance, an improvement ad App Domain level (`AppDelegate` level) there won't be any need of specifying nor commenting on anything.

Something like an extension of `TreeCounter` containing a function:

```
extension TreeCounter {
    public static func initializeAndFetch() {
        _ = TreeCounter.shared
    }
}
```

Reply:
Totally agree this could be much improved, I'm thinking on alternatives together with the new financial reports and will rethink naming as well 🙏

Reply:
Updated with the new Financial reports, let me know what you think now 🙂

Reply:
Checked and indeed is much cleaner to me as least